hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
Add more tests to test the error functionality. 

i.e. if currentStock is 0, expect "Currently unavailable" to be in document.
if isError = true, expect "Some error message" to be in document.
Change the dropdown, expect whatever number to be in document

Reply:
I think I added this in "added the test, still missing line 70" commit that I made right after this if you want to take a look at that